assets/js/pages/ClusterDetails/SBDDetails.jsx

```diff
@@ -10,20 +10,24 @@ const getStatusPill = (status) =>
   );
 
 function SBDDetails({ sbdDevices }) {
+  const sbdDetailsHeader = 'SBD/Fencing';
```

Comment:
This consts can go outside the function, module level (or even in the html tags itself, as they are not repeated)